src/main/java/org/adridadou/ethereum/values/EthAddress.java

```diff
@@ -45,7 +45,9 @@ public static EthAddress of(ECKey key) {
     }
 
     public static EthAddress of(final String address) {
-        if (address != null && address.startsWith("0x")) {
+        if(address==null) 
+        	return empty();
+    	if (address != null && address.startsWith("0x")) {
```

Comment:
No this if (address != null && address.startsWith("0x")) {....
has moved down (to line 50), web3j will return null when we create a contract.

    public static EthAddress of(final String address) {
    	if(address==null)
    		return empty();

    	if (address != null && address.startsWith("0x")) {
            return of(Hex.decode(address.substring(2)));
        }
        return of(Hex.decode(address));
    }